diff -urN torcs-1.2.3.new/Make-config.in torcs-1.2.3/Make-config.in
--- torcs-1.2.3.new/Make-config.in	2005-06-28 12:30:34.000000000 +0200
+++ torcs-1.2.3/Make-config.in	2005-06-29 23:14:39.000000000 +0200
@@ -54,10 +54,10 @@
 #EXT_LIBS = -lpthread @LIBS@
 EXT_LIBS = @LIBS@
 #LDFLAGS =  @LDFLAGS@ -L${EXPORTBASE}/lib -L/usr/lib -L/usr/local/lib
-LDFLAGS =  -L${EXPORTBASE}/lib @LDFLAGS@ -L/usr/lib
+LDFLAGS =  -L${EXPORTBASE}/lib @LDFLAGS@ -L/usr/lib -L/usr/X11R6/lib
 
 #INCFLAGS =  -I${EXPORTBASE}/include -I/usr/include -I/usr/local/include
-INCFLAGS =  -I${EXPORTBASE}/include -I${TORCS_BASE}
+INCFLAGS =  -I${EXPORTBASE}/include -I${TORCS_BASE} -I/usr/X11R6/include
 
 CPP = @CPP@ @CPPFLAGS@ -M -D__DEPEND__
 
